x-pack/plugins/observability_solution/investigate/common/types.ts

```diff
@@ -5,17 +5,14 @@
  * 2.0.
  */
 
-import type { EuiThemeComputed } from '@elastic/eui';
-import type { Filter } from '@kbn/es-query';
-import type { DeepPartial, PickByValue } from 'utility-types';
 import type { AuthenticatedUser } from '@kbn/core/public';
+import type { DeepPartial } from 'utility-types';
 
 export interface GlobalWidgetParameters {
   timeRange: {
     from: string;
     to: string;
   };
-  filters: Filter[];
```

Comment:
We don't need this filter anymore?

Reply:
i removed it indeed because there is no filters bar in the current UI. It wasn't used currently